<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartDialog</class>
 <widget class="QDialog" name="StartDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QListWidget" name="listWidget">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="createPushButton">
         <property name="text">
          <string>Создать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletePushButton">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="openPushButton">
         <property name="text">
          <string>Открыть</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="helpPushButton">
         <property name="text">
          <string>Справка</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
